quickfixj-core/src/test/java/quickfix/SocketInitiatorTest.java

```diff
@@ -145,7 +145,7 @@ public void testBlockLogoffAfterLogon() throws Exception {
                 clientApplication.stopAfterLogon(initiator);
                 clientApplication.setUpLogonExpectation();
 
-                initiator.block();
+                initiator.start();
```

Comment:
I think it does not make sense to change `block()` to `start()` here since the test specifically tests for `block()` (although test is ignored). Maybe mark it as deprecated so it can be cleaned up when the `block()` implementation is cleaned up.